packages/spark/components/_masthead.scss

```diff
@@ -37,6 +37,17 @@
   transform: $sprk-masthead-translateY;
   transition: $sprk-masthead-transition;
 }
+// TODO: Improve --hidden modifier
```

Comment:
i think you should remove the comment block and put it in an issue instead

Reply:
@RVMendoza What are your thoughts? I kind of agree that this should be in the issue.

Reply:
took it out, will add it to issue